src/propgrid/editors.cpp

```diff
@@ -742,6 +742,8 @@ void wxPropertyGrid::OnComboItemPaint( const wxPGComboBox* pCb,
                                        int flags )
 {
     wxPGProperty* p = GetSelection();
+    if (!p) return;
```

Comment:
This seems unrelated, so it'd be be better to extract this change in a separate commit and explain when exactly does this problem happen in the commit message.

Reply:
Submitted https://github.com/wxWidgets/wxWidgets/pull/1008